desktop/renderer/index.js

```diff
@@ -27,6 +27,8 @@ import merge from 'lodash/merge'
 ipcLogsRenderer()
 
 if (module.hot) {
+  debugCancelLogin()
```

Comment:
we try and cancel login if we hot reload, else we'll be 'stuck' in a login rpc and the damon needs to be rebooted, super annoying